chains/evm/calls/deploy.go

```diff
@@ -70,7 +71,7 @@ func deployContract(client ChainClient, abi abi.ABI, bytecode []byte, txFabric T
 	if err != nil {
 		return common.Address{}, err
 	}
-	tx := txFabric(n.Uint64(), nil, big.NewInt(0), config.DefaultGasLimit, gp, append(bytecode, input...))
+	tx := txFabric(nil, n.Uint64(), nil, big.NewInt(0), config.DefaultGasLimit, gp, nil, nil, append(bytecode, input...))
```

Comment:
why its nil?

Reply:
I changed the `TxFabric` to take in chainId which is not needed for the old way we constructed txs, so I just passed it in as `nil` for now, as to not make another client call.